rollup.config.mjs

```diff
@@ -1,48 +1,40 @@
-// The following is only possible from Node 18 onwards
-// import pkg from "./package.json" assert { type: "json" };
+//
+// Copyright Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+import createConfig from "@inrupt/base-rollup-config";
+import pkg from "./package.json" assert { type: "json" };
 
-// Until we only support Node 18+, this should be used instead
-// (see https://rollupjs.org/guide/en/#importing-packagejson) 
-import { createRequire } from 'node:module';
-const require = createRequire(import.meta.url);
-const pkg = require('./package.json');
```

Comment:
We haven't officially removed support for Node 16 yet.

Reply:
Yes - but we only need node 18 to run the build step. This doesn't affect whether our packaged library supports node 16.

Reply:
That is correct, but the CI runs the build step on all supported versions. We should consider building the code once, and then running the tests in the different target runtime, but our code isn't structured to support this so far.

Reply:
It actually appears to be running on node 16 fine (https://github.com/inrupt/solid-client-vc-js/actions/runs/6856123821/job/18642653808?pr=863); looks like it was backported to 16.15.0 (https://stackoverflow.com/questions/73824694/node-16-is-the-experimental-json-modules-cli-flag-still-required).

Reply:
Ah ok, I didn't realize that, no problem then.